pushover_open_client/pushover_open_client.py

```diff
@@ -240,10 +245,19 @@ def writeConfig(self, configFile):
                 'deviceID': self.deviceID,
                 'userID': self.userID
                 }, outfile, indent=4)
-        
-    def login(self):
+
+    def login(self, force=False):

```

Comment:
Curious about the need of this `force` variable on `login` and `registerDevice`? Do you mind enlightening me to its function?

Reply:
`login` should not be doing a real logic if a secret already exists in the config. Same with `registerDevice`. I adjusted the code to check for `secret` and `deviceID` within the config. If those are present, they will simply return successfully without actually performing a login. However, there might be cases in which you'd want login to do a real login, even though there might be a `secret` set in the config. For such cases you can now call `login(force=true)` - as with `registerDevice`.

Reply:
Okay cool, that sounds reasonable